### PR TITLE
fix: add prefix to call to compareCommitsWithBasehead

### PR DESCRIPTION
**Pull request by galenwarren**
To resolve [issue #31](https://github.com/ietf-tools/semver-action/issues/31).

This adds ${prefix} into the call to compareCommitsWithBasehead.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const _ = require('lodash')
 const cc = require('@conventional-commits/parser')
 const semver = require('semver')
 
-async function main () {
+async function main() {
```
Comment (NGPixel) [inline index.js:7]: Don't change the code existing formatting, which follows Standard JS. Same for line 26.

**Review by NGPixel — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (galenwarren) [reply index.js]: Sorry about that -- I didn't mean to change that. My editor did it automatically :). Should be back to the way it was.

Comment (galenwarren) [reply index.js]: @NGPixel Ah, oops. Forgot the build step ... hold on a minute.

Comment (galenwarren) [reply index.js]: @NGPixel Fixed now.

**Review by NGPixel — APPROVED** [review]

**Comment by NGPixel** [comment]
Thanks!

**Comment by galenwarren** [comment]
@NGPixel Thanks!

**Pull request merged**